opennms-webapp/src/main/webapp/includes/help-contact.jsp

```diff
@@ -0,0 +1,55 @@
+<%--
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2002-2017 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2017 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+--%>
+
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <h3 class="panel-title">Get in Contact</h3>
+    </div>
+    <div class="panel-body">
+        <table class="table">
+            <tr>
+                <td style="border-top: none;"><a href="https://www.opennms.com/support/" target="_blank" class="btn btn-default" role="button" style="width: 100%">Commercial Support</a></td>
+                <td style="border-top: none;">Get commercial support for OpenNMS software provided by The OpenNMS Group, Inc.</td>
+            </tr>
+            <tr>
+                <td style="border-top: none;"><a href="https://chat.opennms.org/opennms" target="_blank" class="btn btn-default" role="button" style="width: 100%">Web Chat</a></td>
+                <td style="border-top: none;">Use the web based chat platform powered by Mattermost. The Discussion channel is a good starting point to reach users and developers. If you prefer IRC, you can join the "#opennms" channel on irc.freenode.net which is bridged to our Web Chat.</td>
+            </tr>
+            <tr>
+                <td style="border-top: none;"><a href="https://wiki.opennms.org/wiki/Mailing_lists" target="_blank" class="btn btn-default" role="button" style="width: 100%">Mailing Lists</a></td>
+                <td style="border-top: none;">Get into contact with OpenNMS users in our community through our topic-driven mailing lists. Subsribing the opennms-discuss list is a good starting to ask question and how to get started with OpenNMS.</td>
```

Comment:
"Subsribing the opennms-discuss list is a good starting to ask question and how to get started with OpenNMS." -> "The opennms-discuss mailing list is a good way to ask questions about getting started with OpenNMS."